modelerfour/test/scenarios/body-datetime-rfc1123/modeler.yaml

```diff
@@ -46,7 +46,7 @@ schemas: !<!Schemas>
         namespace: Api100
     protocol: !<!Protocols> {}
   constants:
-  - !<!ConstantSchema> &ref_6
```

Comment:
Does this mean the ConstantSchema isn't being referenced by anyone anymore?  Should it be excluded from the output entirely if the idea is that the response should refer to the underlying value type?

Reply:
Yeah, it's not referenced;  Hmmm. I should probably figure out a way to remove those. 

Other than walking thru the model and ref-counting, I'm not sure I can conclusively know that.